lib/govuk_publishing_components/presenters/machine_readable/faq_page_schema.rb

```diff
@@ -24,9 +24,8 @@ def main_entity
       end
 
       def questions_and_answers_markup
-        question_and_answers(page.body)
-          .select { |_, value| value[:answer].present? }
-          .map do |question, value|
+        question_and_answers.select { |_, value| value[:answer].present? }
+                            .map do |question, value|
```

Comment:
Changed the spacing here so that it was easier to see where the chained calls ended and block code began.